### ISSUE TITLE
Generate XML with skipped testcases

### ISSUE THREAD

**Issue by adrianraszka** [michaelleeallen/mocha-junit-reporter#173]
In Cypress with BDD I can tag scenarios with @skip, what makes them counted as pending specs. 
However mocha-junit-reporter does not list in XML those scenarios, even tho they are counted in <testsuties skipped="1"> 

Im trying to get from test result this:
`<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="3.4040" tests="2" failures="0" skipped="1">
  <testsuite name="Root Suite" timestamp="2022-08-18T14:17:14" tests="0" file="cypress/scenarios/ParaBank/ParaBank_Navigation.feature" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="User can use navigation buttons" timestamp="2022-08-18T14:17:14" tests="3" time="3.4040" failures="2">
    <testcase name="User can use navigation buttons @skip" time="0.0000" classname="@skip">
      <skipped></skipped>
    </testcase>
    <testcase name="User can use navigation buttons PASSED" time="3.4040" classname="PASSED">
    </testcase>
  </testsuite>
</testsuites>`

Not this:

`<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="8.5680" tests="2" failures="0" skipped="1">
  <testsuite name="Root Suite" timestamp="2022-08-18T14:39:02" tests="0" file="cypress/scenarios/ParaBank/ParaBank_Navigation.feature" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="User can use navigation buttons" timestamp="2022-08-18T14:39:02" tests="3" time="7.9800" failures="1">
    <testcase name="User can use navigation buttons PASSED" time="3.4600" classname="PASSED">
    </testcase>
  </testsuite>
</testsuites>`

I need it to properly separate in TestRail reports passed, failed, and not implemented tests.

**Comment by bitcoder** [comment]
I agree. Skipped tests should be part of the report. I can try having a look to see where that may be missing when I have a bit more time

**Comment by bitcoder** [comment]
Have you tried setting the `includePending` option as `true`?

**Comment by adrianraszka** [comment]
I don't have anymore any code to check your solution.  Did it worked for you? I could try testing it on the weekend.

**Comment by debajitdruva** [comment]
I am seeing similar issues. In my mochawesome report there are separate object for the skipped one like below.
```javascript
        {
              "title": "Testcase title",
              "fullTitle": "Spec title Testcase title",
              "timedOut": null,
              "duration": 0,
              "state": "skipped",
              "speed": null,
              "pass": false,
              "fail": false,
              "pending": false,
              "context": null,
              "code": "....failed code snippet",
              "err": {},
              "uuid": "722c31f6-8bd9-4037-8b44-22a39ea67396",
              "parentUUID": "3522455b-b6ba-4618-98ad-41c00c383b06",
              "isHook": false,
              "skipped": true
            }
```

However, there is no corresponding entry in the junit xml report. Tried with `includePending:true` as well but no luck

**Comment by debo07** [comment]
I am seeing similar issues. In my mochawesome report there are separate object for the skipped one like below.
```javascript
        {
              "title": "Testcase title",
              "fullTitle": "Spec title Testcase title",
              "timedOut": null,
              "duration": 0,
              "state": "skipped",
              "speed": null,
              "pass": false,
              "fail": false,
              "pending": false,
              "context": null,
              "code": "....failed code snippet",
              "err": {},
              "uuid": "722c31f6-8bd9-4037-8b44-22a39ea67396",
              "parentUUID": "3522455b-b6ba-4618-98ad-41c00c383b06",
              "isHook": false,
              "skipped": true
            }
```

However, there is no corresponding entry in the junit xml report. Tried with `includePending:true` as well but no luck
